packages/react/src/Button/Button.features.stories.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react'
+import {ComponentMeta} from '@storybook/react'
+import {userEvent, waitFor, within} from '@storybook/testing-library'
+
+import {Button} from '.'
+
+export default {
+  title: 'Components/Button/Features',
+  component: Button
+} as ComponentMeta<typeof Button>
+
+export const Primary = () => (
+  <Button as="a" variant="primary" size="medium" href="#">
+    Primary action
+  </Button>
+)
+
+export const PrimaryDisabled = () => (
+  <Button as="a" disabled variant="primary" href="#">
+    Disabled primary button
+  </Button>
+)
+
+export const PrimaryFocus = () => (
+  <Button variant="primary" size="large" href="#">
+    Focus
+  </Button>
+)
+PrimaryFocus.play = async () => {
+  await waitFor(async () => {
+    userEvent.tab()
+  })
+}
+
+export const PrimaryFocusNonStandardBG = () => (
```

Comment:
This one shows what the focus state looks like on backgrounds that _don't_ use --brand-color-canvas-default color. As you'll see it applies a canvas color here, which is required as transparent isn't a valid color and we need to add _something_ that provides a buffer between the button and focus ring.

🔗 https://primer-0ed242c419-26139705.drafts.github.io/storybook/?path=/story/components-button-features--primary-focus-non-standard-bg

Reply:
@ashygee let me know if this is okay 🙏